scripts/build_wheel.py

```
@@ -380,11 +380,17 @@ def check_missing_libs(lib_name: str) -> list[str]:
 
-def generate_python_stubs_linux(venv_python: Path, deep_ep: bool,
-                                flash_mla: bool, transfer_agent_binding: bool,
-                                binding_lib_name: str):
-    build_run(f"\"{venv_python}\" -m pip install nanobind")
-    build_run(f"\"{venv_python}\" -m pip install pybind11-stubgen")
+def generate_python_stubs_linux(venv_python: Path,
+                                deep_ep: bool,
+                                flash_mla: bool,
+                                transfer_agent_binding: bool,
+                                binding_lib_name: str,
+                                constraint_file: Path = None):
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)